tools/reviewer/web/src/app/dashboard/diff/diff-discussion/diff-discussion.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input } from '@angular/core';
+
+import { Comment, Thread } from '@/shared/proto';
+import { AuthService } from '@/shared/services';
+import { DiffService } from '../diff.service';
+
+@Component({
+  selector: 'diff-discussion',
+  templateUrl: './diff-discussion.component.html',
+  styleUrls: ['./diff-discussion.component.scss'],
+  providers: [DiffService],
+})
+export class DiffDiscussionComponent {
```

Comment:
Can you please document what each component is?
One way to do it is to add a screenshot of the webapp with a rectangle on the component (if it's a visible component). We can host the image in some image hosting service of your choice (or Firebase hosting even).

Reply:
I'm not sure about the idea with screenshots. I added them, but I think it's difficult to maintain the screenshots up to date.

Reply:
It's true. It's just to get a basic understanding of where the component is on the page. Might have been an illustration or sketch, but it's easier to take a screenshot.

Reply:
It's 331.6 kB, but ok.